.github/workflows/release.yml

```diff
@@ -42,6 +39,9 @@ jobs:
         with:
           python-version: "3.9.5"
 
+      - name: Get dbt-trino version
+        run: echo "version_number=$(cat .bumpversion.cfg | grep current_version | cut -d " " -f3)" >> $GITHUB_ENV
```

Comment:
this is rather hacky :) 

any way to get the current version in a more elegant way?

Reply:
that's an official way of handling this: https://docs.github.com/en/actions/learn-github-actions/workflow-commands-for-github-actions#setting-an-environment-variable